table/tables/testutil.go

```diff
@@ -16,6 +16,7 @@ package tables
 
 import "github.com/pingcap/tidb/table"
 
+// SwapReorgPartFields swaps the reorganizePartitions field of two partitioned tables. used in tests.
 func SwapReorgPartFields(src, dst table.Table) bool {
```

Comment:
Moving from a `*_test.go` file pattern to `testutil.go`, would not that mean that this function now also gets exported in non-test context?

Reply:
It is possible for other developers to misuse the function. the best method is to move it into the ```partition_test```. but I found it needs to change more private functions into public.

Reply:
The most important thing is to make ```partitionedTable``` public.